prime-router/src/test/kotlin/azure/ActionHistoryTests.kt

```diff
@@ -1,5 +1,7 @@
 package gov.cdc.prime.router.azure
 
+import assertk.assertThat
+import assertk.assertions.* // ktlint-disable no-wildcard-imports
```

Comment:
Why disable.  Let the IDE handle these and not use wildcards.

Reply:
Agree with @carlosfelix2. We don't do wildcard imports. Please fix this.

Reply:
I agree.

Reply:
Follow up: made the changes.

Reply:
Follow up: made the changes.